internal/syscall/syscall_linux.go

```diff
@@ -32,11 +32,13 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
+var logger = grpclog.Component("TRANSPORT")
```

Comment:
`transport` doesn't sound right here. Leave it in `core`?

Reply:
Change the syscall logger to CORE